Makefile

```
@@ -25,33 +25,20 @@ VALEOPTS        ?=
 help:  # This help message
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-# environment management
-.venv/bin/python:  ## Create Python virtual environment and install package requirements
-	python3 -m venv .venv
-	.venv/bin/pip install --upgrade pip setuptools uv wheel
-	.venv/bin/uv pip install -r requirements.txt
-
-.PHONY: docs
-docs: .venv/bin/python  ## Install documentation requirements into Python virtual environment
-	.venv/bin/uv pip install -r requirements-docs.txt -e .
 
+# environment management
 .PHONY: dev
-dev: docs  ## Install development requirements into Python virtual environment
-	.venv/bin/uv pip install -r requirements-dev.txt -e .
-
-.PHONY: update-deps
-update-deps:  ## Update
-	.venv/bin/uv pip install --upgrade pip setuptools uv wheel
-	.venv/bin/uv pip compile --upgrade pyproject.toml -o requirements.txt
-	.venv/bin/uv pip compile --upgrade pyproject.toml \
-		--extra docs -o requirements-docs.txt
-	.venv/bin/uv pip compile --upgrade pyproject.toml \
-		--extra docs --extra dev -o requirements-dev.txt
+dev:  ## Install required Python, create Python virtual environment, and install package requirements
+	@uv python install ">=3.11,<3.13"
```

```suggestion
	@uv python install ">=3.10,<3.14"
```

ref: https://plone-sphinx-theme.readthedocs.io/guides/usage.html#prerequisites

---

Whoops, actually this should be:

```make
@uv python install ">=3.11,<3.14"
```

Why the difference? Usage != Development.

For users of the theme, they can still use Python 3.10, such as Plone 6.0 projects. That's handled in `pyproject.toml` when the user's project resolves package dependencies upon installation. Python 3.10 installs an older yet compatibile version of Sphinx which lacks a few nice features that appeared in Sphinx 8.2.x, but are not absolutely required. See https://www.sphinx-doc.org/en/master/changes/8.2.html#release-8-2-0-released-feb-18-2025

For contributors to the theme, the Python version is handled by `make dev`. I debated pushing the lower bound to Python 3.13, but I think allowing a range of Pythons allows us to catch issues that we might miss. In fact, I think I should add a Python version to the matrix in CI to do exactly that.

---

Done in https://github.com/plone/plone-sphinx-theme/pull/67/commits/81042715480091cae9ab3f788acc4b48c601f4a9 and https://github.com/plone/plone-sphinx-theme/pull/67/commits/1fde842d105ea148a594d2a0d786f7b758d183c6

---

after retrying the failed jobs, they actually passed! thanks @stevepiercy

---

Well, they passed, but the Python 3.10 job was using 3.13. I sorted that out in subsequent commits. I'm going to merge it now.